src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/PetFeatures.kt

```diff
@@ -53,22 +53,6 @@ object PetFeatures {
     private var lastPetLockNotif: Long = 0
     var lastPet: String? = null
 
-
```

Comment:
uhhh I can think of a case where this is useful, cause maybe hypixel hide pet nametags might mess up with some mod feature or something idek

Reply:
doubt a mod uses pet nametags for anything since most people have pets hidden anyways